### PR TITLE
A11y for scrollnode

### PR DESCRIPTION
**Pull request by wsdwsd0829**

### CODE REVIEW

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Any source code changes should have an entry in CHANGELOG.md or have #trivial in their title.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by wsdwsd0829** [comment]
becuse ASScollView is not backed by _ASDisplayView but UIScrollView.

File: Source/Details/_ASDisplayViewAccessiblity.mm [hunk 1]
```diff
@@ -271,4 +271,19 @@ - (NSArray *)accessibilityElements
 
 @end
 
+@implementation ASDisplayNode (AccessibilityInternal)
+
+- (NSArray *)accessibilityElements {
```
Comment (maicki) [inline Source/Details/_ASDisplayViewAccessiblity.mm]: Braces on new line

File: Source/ASScrollNode.mm [hunk 2]
```diff
@@ -52,6 +52,10 @@ - (void)didMoveToWindow
   }
 }
 
+- (NSArray *)accessibilityElements {
```
Comment (maicki) [inline Source/ASScrollNode.mm]: Braces on newline

File: Source/Details/_ASDisplayViewAccessiblity.mm [hunk 3]
```diff
@@ -271,4 +271,19 @@ - (NSArray *)accessibilityElements
...
+  if (!self.isNodeLoaded) {
```
Comment (maicki) [inline Source/Details/_ASDisplayViewAccessiblity.mm]: Can we not just call thought to the views `accessibilityElements`?

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/45842b901ebe2fbf2a10aea82e76dc96/raw/a1334eebe43c9a5746de997a594958c0b0753ca3/log.txt)

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/b165a9819fe37b276fafb5cec359d4c8/raw/f508a95e70dd9a411a07009e1e493b686fabde79/log.txt)

**Comment by maicki** [comment]
@wsdwsd0829 The test will not work if the accessibility inspector is not running (or was at least launched for the simulator we are testing on once). The reason is that calling `[subnode accessibilityElementCount]` that will call the same method on the view of the node will call through to `-[ASDisplayNode accessibilityElements]` *only* if accessibility is enabled. Otherwise it will just return 0.

We had the same issue with #1069 a while back. I would recommend handling it the same way we did in #1069 and disable the lines that are failing currently.

**Comment by maicki** [comment]
Thanks @wsdwsd0829 for working on this!

**Pull request merged**